fdbserver/workloads/Mako.h

```diff
@@ -0,0 +1,33 @@
+#ifndef MAKO_H
```

Comment:
Is there a reason that this exists in its own header-file? I don't think this will ever make sense to be included anywhere else than mako.actor.cpp - therefore I would just copy&paste the contents into this file...

Reply:
I agree. will move it.